techmeet/data/models/user.py

```diff
@@ -0,0 +1,67 @@
+"""Custom user model."""
+from typing import Any
+
+from django.contrib.auth.base_user import BaseUserManager
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+
+class UserManager(BaseUserManager):
+    """User manager that uses email as login."""
+
+    def _create_user(
+        self, name: str, email: str, password: str, **extra_fields: Any
+    ) -> "User":
+        """Create and save a User with the given email and password."""
+        if not email:
+            raise ValueError(_("Email must be set"))
+        if not name:
+            raise ValueError(_("Name must be set"))
+        email = self.normalize_email(email)
+        user = self.model(email=email, name=name, **extra_fields)
+        user.set_password(password)
+        user.save()
+        return user
+
+    def create_user(
+        self, name: str, email: str, password: str, **extra_fields: Any
```

Comment:
I really like using `extra_fields` rather than `kwargs`

Reply:
Same pattern as django uses in their code.